README.md

```diff
@@ -1,61 +1,21 @@
-# pvsadm
-
 ## Overview
-Power Systems Virtual Server projects deliver flexible compute capacity for Power Systems workloads.
-Integrated with the IBM Cloud platform for on-demand provisioning.
-
-This is a tool built for the Power Systems Virtual Server helps managing and maintaining the resources easily
 
-```shell script
-./pvsadm --help
-Power Systems Virtual Server projects deliver flexible compute capacity for Power Systems workloads.
+IBM Power Systems Virtual Server projects deliver flexible compute capacity for Power Systems workloads.
 Integrated with the IBM Cloud platform for on-demand provisioning.
 
-This is a tool built for the Power Systems Virtual Server helps managing and maintaining the resources easily
-
-Usage:
-  pvsadm [command]
-
-Available Commands:
-  help        Help about any command
-  purge       Purge the powervs resources
-
-Flags:
-  -k, --api-key string   IBMCLOUD API Key(env name: IBMCLOUD_API_KEY)
-      --debug            Enable PowerVS debug option
-  -h, --help             help for pvsadm
-
-Use "pvsadm [command] --help" for more information about a command.
-```
-
-## Purge PowerVS Resources:
-
-```shell script
-$ ./pvsadm purge --help
-Purge the powervs resources
-
-Usage:
-  pvsadm purge [command]
+This is a tool built for the IBM Power Systems Virtual Server helps managing and maintaining the resources easily.
```

Comment:
This is a tool to help with managing of resources in IBM Power Systems Virtual Server

Reply:
Completed.